docs/ja/core-concepts/ngsiv2-vs-ngsild.md

```
@@ -5,27 +5,38 @@ outline: deep
-- [プロトコルアイソレーション](#プロトコルアイソレーション)
-- [属性タイプマッピングテーブル](#属性タイプマッピングテーブル)
-- [システム属性の違い](#システム属性の違い)
-- [出力フォーマットの違い](#出力フォーマットの違い)
-- [共有機能](#共有機能)
-- [NGSI-LD 固有の機能](#ngsi-ld-固有の機能)
-- [エンティティ ID の考慮事項](#エンティティ-id-の考慮事項)
-- [フェデレーション](#フェデレーション)
-- [ユースケースとベストプラクティス](#ユースケースとベストプラクティス)
 
----
+* [概要](#overview)
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)